### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by buddizer**
- Removed remaining hard coded phrases and added them to en.lua/es.lua
- Added code for server.cfg locale convar auto changing locale

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -7,10 +7,10 @@ Config.MinimumStress = 50 -- Minimum Stress Level For Screen Shaking
 Config.MinimumSpeedUnbuckled = 50 -- Going Over This Speed Will Cause Stress
 Config.MinimumSpeed = 100 -- Going Over This Speed Will Cause Stress
 Config.DisablePoliceStress = false -- Default: false, If true will disable stress for people with the police job
-Config.FuelScript = 'LegacyFuel' -- change to lj-fuel if you use lj-fuel or something else if you use any other LegcyFuel compatible script
+Config.FuelScript = 'ps-fuel' -- change to lj-fuel if you use lj-fuel or something else if you use any other LegcyFuel compatible script
```
Comment (MonkeyWhisper) [inline config.lua]: Set back to LegacyFuel

File: config.lua [hunk 2]
```diff
@@ -7,10 +7,10 @@ Config.MinimumStress = 50 -- Minimum Stress Level For Screen Shaking
...
 
 -- Admin only to change hud icons/shapes
-Config.AdminOnly = false
+Config.AdminOnly = true
```
Comment (MonkeyWhisper) [inline config.lua]: Set back to false

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: fxmanifest.lua [hunk 3]
```diff
@@ -2,11 +2,12 @@ fx_version 'cerulean'
 game 'gta5'
 
 description 'ps-hud'
-version '2.1.0'
```
Comment (MonkeyWhisper) [inline fxmanifest.lua:5]: We never really did a version, but I just released a 2.0, can you do add it as a 2.1? 

Thank you!

Comment (buddizer) [reply fxmanifest.lua]: Updated

File: config.lua (hunk 1 above)
Comment (buddizer) [reply config.lua]: Yup, already done, it was sent in the commit by mistake

File: config.lua (hunk 2 above)
Comment (buddizer) [reply config.lua]: Yup, already done, it was sent in the commit by mistake

**Pull request merged**